java/Dockerfile.tpl

```diff
@@ -44,6 +44,22 @@ RUN echo "Starting ..." && \
     apt-get -qq -y install maven && \
     echo "Done Install Maven!" && \
 
+    echo "Install graphviz" && \
+    apt-get -qq -y install \
+        libcdt5 \
+        libcgraph6 \
+        libgd3 \
+        libgvc6 \
+        libgvpr2 \
+        libxaw7 \
+        libxmu6 \
+        libxt6 \
+        libann0 && \
+    curl http://ftp.debian.org/debian/pool/main/g/graphviz/graphviz_2.38.0-17_amd64.deb --output graphviz.deb && \
```

Comment:
You should add an argument to manage the version

Reply:
And maybe use the Debian's package?

Reply:
Done